ynput/OpenPype
# AYON: Thumbnails cache and api prep

## Changelog Description
Moved thumbnails cache from ayon python api to OpenPype and prepare AYON thumbnail resolver for new api functions. Current implementation should work with old and new ayon-python-api.

## Additional info
The caching of thumbnails does not belong to ayon python api ([in progress PR](https://github.com/ynput/ayon-python-api/pull/83)). The PR also implements `get_thumbnail_by_id` which can be used in current thumbnail resolver instead.

## Testing notes:
1. Make sure you have a folder or product with thumbnail on AYON server
2. Open loader from tray
3. Select the same folder or product
4. Thumbnail should show up
